client/my-sites/hosting/github/github-authorize-card/index.tsx

```diff
@@ -0,0 +1,39 @@
+import { Button, Card } from '@automattic/components';
+import requestExternalAccess from '@automattic/request-external-access';
+import { translate } from 'i18n-calypso';
+import { useSelector, useDispatch } from 'react-redux';
+import CardHeading from 'calypso/components/card-heading/index';
+import { requestKeyringConnections } from 'calypso/state/sharing/keyring/actions';
+import { getKeyringServiceByName } from 'calypso/state/sharing/services/selectors';
+import iconGitHub from '../github.svg';
+
+import './style.scss';
+
+type Service = {
```

Comment:
Another option is to define this type in the selector file. That way you're improving it everywhere.

https://github.com/Automattic/wp-calypso/blob/45d16bd8af006ea17a6c29d414dd770f29b4b21d/client/state/sharing/services/selectors.js#L37

You'll need to use JSDoc though since it's a JS file.

```js
/**
 * @typedef {Object} Service
 * @property {string} connect_URL
 */
```